README.md

```diff
@@ -187,6 +187,38 @@ class PostSerializer < ActiveModel::Serializer
 end
 ```
 
+### Overriding the Key Format
+
+You can specify that serializers use a different style for the format of the
+keys that are sent over. This can be done globally or at the controller action
+level.
+
+Each adapter has a default format, so typically you're either not going to have
+to change this, or you'll edit the global setting.
+
+For example, the JSON adapter defaults to `lower_camel` whereas the JSON API
+adapter defaults to `dasherize`. Any other adapter will default to `unaltered`.
+
```

Comment:
@beauby I am very much in the other camp on that.  The JSON API standard recommends dasherized keys and the couple JSON API JS libraries I've seen default to expecting _incoming_ requests to be formatted with dasherized keys.

In the vein of making this Just Work with JSON API, I think it should absolutely be dasherized.  Otherwise almost everyone who uses it will need to add the global config option to their projects.

I'll bow to the majority opinion, but I just wanted to voice my concern.